datafusion/sqllogictest/test_files/errors.slt

```diff
@@ -120,7 +120,7 @@ from aggregate_test_100
 order by c9
 
 # WindowFunction with BuiltInWindowFunction wrong signature
-statement error DataFusion error: Error during planning: No function matches the given name and argument types 'NTH_VALUE\(Int32, Int64, Int64\)'\. You might need to add explicit type casts\.\n\tCandidate functions:\n\tNTH_VALUE\(Any, Any\)
+statement error Coercion
 select
 c9,
 nth_value(c5, 2, 3) over (order by c9) as nv1
```

Comment:
I deleted `Coercion` so that test will fail. Then I used the `--complete` flag to auto-generate the multiline error statement automatically.

`RUST_BACKTRACE=1 cargo test -p datafusion-sqllogictest --test sqllogictests errors.slt -- --complete`

```suggestion
statement error
select
c9,
nth_value(c5, 2, 3) over (order by c9) as nv1
from aggregate_test_100
order by c9
----
DataFusion error: Error during planning: Error during planning: Coercion from [Int32, Int64, Int64] to the signature OneOf([Any(0), Any(1), Any(2)]) failed. No function matches the given name and argument types 'nth_value(Int32, Int64, Int64)'. You might need to add explicit type casts.
	Candidate functions:
	nth_value()
	nth_value(Any)
	nth_value(Any, Any)
```

Reply:
I applied this suggestion but now sure why it remained like this. Resolving it. Thanks